Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -7,11 +7,22 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 Due to package verification, the latest version below is the unpublished version and the date is meaningless.
 however, it has to be formatted properly to pass verification tests.
 
+## [1.0.0-preview2] - 2019-?-??
```

Comment:
Is `preview` followed by `preview2` or `preview1`? (I have been using `preview1` in my currently open PRs). Also, you will need to update your package.json to get tests green likely.

Reply:
Hmm, if the `com.unity.package-manager-doctools` package we are using uses the scheme correctly, then we are both doing it wrong, and it should be: `1.0.0-preview.1` (with a dot)

Reply:
I will fix it correctly (using -preview.1) in https://github.com/Unity-Technologies/InputSystem/pull/885, then we can land that and merge it to the other PRs

Reply:
>I will fix it correctly (using -preview.1) in #885, then we can land that and merge it to the other PRs

Awesome.

>Hmm, if the com.unity.package-manager-doctools package we are using uses the scheme correctly, then we are both doing it wrong, and it should be: 1.0.0-preview.1 (with a dot)

Looking at the semver spec, looks like `preview.1` is indeed the correct way.

>Is preview followed by preview2 or preview1?

Good question. Given that everything else in the version is zero-based, should probably be 1 and not 2.